packages/grid/x-data-grid-generator/src/commodities.columns.tsx

```diff
@@ -1,4 +1,4 @@
-import { gridStringOrNumberComparator } from '@mui/x-data-grid';
+import { gridStringOrNumberComparator } from '@mui/x-data-grid-pro';
```

Comment:
I assume data-grid-generator must only import from DataGridPro to be sure that it can work with both DataGrid and DataGridPro.

Reply:
I migrated all those imports to the pro package to avoid type conflicts and be consistent.